site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -345,6 +345,14 @@ In the support ticket, please provide:
 
 For Business and High Availability deployments, backups from an upgrade are available for 30 days. For Basic deployments, no backups are available.
 
+Both production and non-production environments can be rolled back if on a plan with backups. 
+
+Production rollbacks will be treated the same as a production outage.
+
+Non-production rollbacks will be completed within two business days, and will be handled during normal FusionAuth business hours (9am-5pm Mountain time). Non-production rollbacks will not be treated as a production outage.
```

Comment:
I would add a `M-F` designation for normal business hours.

Reply:
Updated.